src/components/HomePage/Forms/AddMovieForm.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { FC } from 'react';
+import {
+  Button, Form, FormGroup, Input, Label,
+} from 'reactstrap';
+import './Form.css';
+
+export const AddMovieForm: FC = () => {
+  const movieGenres = ['Action', 'Adventure', 'Animation',
+    'Comedy', 'Drama', 'Family',
+    'Fantasy', 'Horror', 'Music',
+    'Mystery', 'Romance', 'Science Fiction',
+    'Thriller'];
+
+  return (
+    <Form id="add-movie-form-id">
+      <FormGroup>
```

Comment:
I suggest adding separate component for formgroup inputs with label.

To replace 
<FormGroup>
        <Label for="releaseDate">
          Release Date
        </Label>
        <Input
          id="releaseDate"
          name="date"
          placeholder="Select Date"
          type="date"
        />
      </FormGroup>

by this:
<InputWithLabel
          id="releaseDate"
          label="Release Date"
          name="date"
          placeholder="Select Date"
          type="date"
/>

Reply:
Fixed in commit 603987e4460f9867aabfe4bcd45a3b7cf3f62548 https://github.com/Olga2157/react-movies-app/commit/603987e4460f9867aabfe4bcd45a3b7cf3f62548